barretenberg/ts/src/main.ts

```diff
@@ -122,7 +158,7 @@ export async function proveAndVerify(bytecodePath: string, witnessPath: string,
 
 export async function proveAndVerifyUltraHonk(bytecodePath: string, witnessPath: string, crsPath: string) {
   /* eslint-disable camelcase */
-  const { api } = await init(bytecodePath, crsPath, -1, true);
+  const { api } = await initUltraHonk(bytecodePath, crsPath, -1);
```

Comment:
isn't the third parameter -1 by default? 
also why exactly is it needed?

Reply:
oops yeah, you're right. I'll delete it